code/model/FreshdeskAgents.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+class FreshdeskAgents extends DataObject
```

Comment:
It's a convention that DataObjects in SS are singularly named (FreshdeskAgent)

Reply:
ack

Reply:
Missed file rename :-)

Reply:
It has been renamed https://github.com/jakedaleweb/silverstripe-freshdesk/pull/6/files#diff-bbd03846b4ad5bc0d608cc72e10753f7R3